pyperformance/venv.py

```diff
@@ -24,6 +24,8 @@ def from_benchmarks(cls, benchmarks):
         for bench in benchmarks or ():
             filename = bench.requirements_lockfile
             self._add_from_file(filename)
+            if bench.install_setup:
+                self._add(os.path.dirname(bench.metafile))
```

Comment:
Along with my other comment:

```suggestion
            if bench.setup_py:
                # pip doesn't support installing a setup.py,
                # but it does support installing from the directory it is in.
                self._add(os.path.dirname(bench.setup_py))
```